src/functions-and-arrays.js

```diff
@@ -1,41 +1,96 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) { return num1 }
+  else { return num2 }
+}
+result = maxOfTwoNumbers(3, 5);
+console.log(result);
+
 
 
 
 // Iteration #2: Find longest word
+
+function findLongestWord(words) {
+
+  let wordLength = [];
+  wordLength.push(words[0]);
+  //console.log(wordLength);
+  //console.log(typeof (wordLength));
+
+  for (let i = 0; i < words.length; i++) {
+    if (wordLength.length < words[i].length) {
+      wordLength[0] = words[i]
+    }
+  } 
+  return wordLength[0]
+}
+
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+let result2 = findLongestWord(words);
+console.log(result2);
 
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(numbers) {
+  let summedUp = 0;
+  
+  for (let i = 0; i < numbers.length; i++) {
+    summedUp = summedUp + numbers[i]
+  }
+  return summedUp
+}
+
+let result3 = sumNumbers(numbers);
+console.log(result3);
 
 
 
 // Iteration #3.1 Bonus:
-function sum() {}
+function sum() { }
 
 
 
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(numbersAvg) {
+  let averageNumbersSum = 0;
+
+  for (let i = 0; i < numbersAvg.length; i++) {
+    averageNumbersSum = averageNumbersSum + numbersAvg[i]
+  }
+  return averageNumbersSum/2
+}
+
+let result4 = averageNumbers(numbersAvg);
```

Comment:
You should divide the total sum by the amount of numbers in the array to get the average